### PR TITLE
fixes_177

### LINKED ISSUE #177: As a developer I need access to the folder icon so that I can implement folders
**Is your feature request related to a problem? Please describe.**
[Weekplanner#223](https://github.com/aau-giraf/weekplanner/issues/223) requires a folder icon.

**Describe the solution you'd like**
Folder icon uploadet to [here](https://github.com/aau-giraf/wiki/blob/master/docs/development/design_guide/icons.md), [here](https://github.com/aau-giraf/wiki/tree/master/docs/development/design_guide/images/Icons%20GIRAF/35x35px) and [here](https://github.com/aau-giraf/wiki/tree/master/docs/development/design_guide/images/Icons%20GIRAF/SVG).

### PR DESCRIPTION
**Pull request by T-Dawq**
fixes #177 
Icon added to:
 - design_guide/icons.md
 - design_guide\images\Icons GIRAF\35x35px
 - design_guide\images\Icons GIRAF\SVG

### CODE REVIEW

File: design_guide/icons.md [hunk 1]
```diff
@@ -30,8 +30,8 @@ If a new icon is added, it should be added to this table.
...
-| Stop               | ![stop](./images/Icons%20GIRAF/35x35px/stop.png)                        | Stops and resets the timer                                                    |
-| Pause               | ![pause](./images/Icons%20GIRAF/35x35px/pause.png)                        | Pauses the timer                                                    |
+| Stop               | ![stop](./images/Icons%20GIRAF/35x35px/stop.png)                         | Stops and resets the timer                                         |
+| Pause              | ![pause](./images/Icons%20GIRAF/35x35px/pause.png)                       | Pauses the timer                                                   |
 | Synced             | TBD                                                                      | Indicates that the current object is syncronized                   |
 | Undo               | ![undo](./images/Icons%20GIRAF/35x35px/undo.png)                         | Reverts latest action                                              |
-
+| Folder             | ![folder](./images/Icons%20GIRAF/35x35px/folder.png)                     | Used for categorizing citizens                                     |
```
Comment (DumbLol) [inline design_guide/icons.md]: Please move to correct line number.
Rest of list is alphabetically sorted.

File: design_guide/icons.md [hunk 2]
```diff
@@ -30,8 +30,8 @@ If a new icon is added, it should be added to this table.
 | Search             | ![search](./images/Icons%20GIRAF/35x35px/search.png)                     | Indicates that it is possible to search for something              |
 | Settings           | ![Settings](./images/Icons%20GIRAF/35x35px/settings.png)                 | Accesses the settings menu                                         |
 | Start              | ![start](./images/Icons%20GIRAF/35x35px/start.png)                       | Starts the timer                                                   |
-| Stop               | ![stop](./images/Icons%20GIRAF/35x35px/stop.png)                        | Stops and resets the timer                                                    |
-| Pause               | ![pause](./images/Icons%20GIRAF/35x35px/pause.png)                        | Pauses the timer                                                    |
+| Stop               | ![stop](./images/Icons%20GIRAF/35x35px/stop.png)                         | Stops and resets the timer                                         |
+| Pause              | ![pause](./images/Icons%20GIRAF/35x35px/pause.png)                       | Pauses the timer                                                   |
```
Comment (DumbLol) [inline design_guide/icons.md:36]: Is there a reason for these white space changes?

**Review by DumbLol — CHANGES_REQUESTED** [review]
What about the add new from the prototype?
What about the icon for add new folder?

File: design_guide/icons.md (hunk 2 above)
Comment (T-Dawq) [reply design_guide/icons.md:36]: They align the vertical line seperating the columns. At least in my text editor.

File: design_guide/icons.md (hunk 1 above)
Comment (T-Dawq) [reply design_guide/icons.md]: Done.

**Review by DumbLol — APPROVED** [review]
looks good to me

**Review by Lortemanden — APPROVED** [review]

**Review by nicolaibjerring — APPROVED** [review]

**Review by karlog100 — COMMENTED** [review]
Looks fine

**Pull request closed without merging**